cmd/snap-repair/runner_test.go

```diff
@@ -1391,3 +1415,256 @@ AXNpZw==`}
 	c.Assert(err, IsNil)
 	c.Check(string(scrpt), Equals, "exit 0\n")
 }
+
+func makeMockRepair(script string) string {
+	return fmt.Sprintf(`type: repair
+authority-id: canonical
+brand-id: canonical
```

Comment:
Should we use `can0nical` as we do in other places with fake-data?

Reply:
I think we could, I leave that to @pedronis to decide.

Reply:
we use either, so far cmd/snap-repair is using canonical everywhere